docs/framework/wpf/getting-started/walkthrough-my-first-wpf-desktop-application.md

```diff
@@ -54,17 +54,19 @@ The first step is to create the application infrastructure, which includes an ap
 
       The **Create a new project** dialog opens.
 
-      ![Create a new project dialog](./media/gettingstartedfigure0a.png)
+      ![Create a new project dialog](./media/walkthrough-my-first-wpf-desktop-application/create-new-project-dialog.png)
 
    2. In the **Language** dropdown, select either **C#** or **Visual Basic**.
 
+      ![New Project dialog with WPF app selected](./media/walkthrough-my-first-wpf-desktop-application/new-project-dialog-box.png)
```

Comment:
this is a VS 2017 screenshot, so it should actually be removed since we're using 2019 here.